.env.example

```diff
@@ -24,9 +24,8 @@ VUE_APP_REGISTRIES_SEARCH_API_URL="https://bcregistry-dev.apigee.net/registry-se
 VUE_APP_REGISTRIES_SEARCH_API_VERSION="/api/v1"
 VUE_APP_REGISTRIES_SEARCH_API_KEY=
 
-
 #vaults launchdarkly
-VUE_APP_NAMEREQUEST_LD_CLIENT_ID=""
+VUE_APP_NAMEREQUEST_LD_CLIENT_ID="5fbec4b5d998e30990020007"
```

Comment:
This isn't really a "secret" (other UIs have this in their repo), and it's handy for devs starting work in this repo that need to copy `.env.example` to `.env`.